make/migrations/postgresql/0050_2.2.0_schema.up.sql

```diff
@@ -268,3 +268,13 @@ BEGIN
   UPDATE scanner_registration SET is_default = TRUE WHERE name = 'Trivy' AND immutable = TRUE;
   END IF;
 END $$;
+
+/* Remove these columns in scan_report because execution-task pattern will handle them */
+ALTER TABLE scan_report DROP COLUMN IF EXISTS job_id;
+ALTER TABLE scan_report DROP COLUMN IF EXISTS track_id;
+ALTER TABLE scan_report DROP COLUMN IF EXISTS requester;
+ALTER TABLE scan_report DROP COLUMN IF EXISTS status;
```

Comment:
It seems `status` is still using.

Reply:
`Status` is still kept in the model of `ScanReport` but it will be filled from the status of the task mode.

```golang
Status    string    `orm:"-"`
StartTime time.Time `orm:"-"`
EndTime   time.Time `orm:"-"`
```